exercises/dominoes/tests/dominoes.rs

```diff
@@ -6,9 +6,9 @@ type Domino = (usize, usize);
 
 #[derive(Debug)]
 enum CheckResult {
-    GotInvalid,             // chain returned None
+    GotInvalid, // chain returned None
     Correct,
-    ChainingFailure(Vec<Domino>), // failure to match the dots at the right side of one domino with
+    ChainingFailure(Vec<Domino>), /* failure to match the dots at the right side of one domino with */
```

Comment:
this one's a little weird, since the comment spans two line and now one line has `/* */` and the next has `//`. I wonder why rustfmt didn't apply the same change to line 14.